samples/ballerina-by-example/examples/kafka_message_consumer_ssl/kafka_message_consumer_ssl.bal

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import wso2/kafka;
+import ballerina/io;
+import ballerina/log;
+import ballerina/encoding;
+
+kafka:ConsumerConfig consumerConfigs = {
+    // Here we create a consumer configs with SSL parameters
+    // securityProtocol - communication protocol
+    // sslTruststoreLocation - location of truststore file
+    // sslTruststorePassword - truststore password
+    // sslKeystoreLocation - location of keystore file
+    // sslKeystorePassword - keystore password
```

Comment:
Better to add the description of parameters before initialise the config.

Reply:
Since this is a sample and these are just comments, we can't move the descriptions before the initialization. If we do so, we have to document the parameter and that is not possible for samples.

Reply:
Since this is a sample and these are just comments, we can't move the descriptions before the initialization. If we do so, we have to document the parameter and that is not possible for samples.

Reply:
Shall we use in-line comments then?